capstone_project/src/main/webapp/style.css

```diff
@@ -1,6 +1,131 @@
-#content {
-  width: 100%;
+html,
+body {
   height: 100%;
   margin: 0;
+  width: 100%;
+  color: #282828 !important;
+  background-color: rgb(94, 191, 204);
```

Comment:
i think the css is sorted, except here. Could you sort them? Also, anywhere else in this file that I might have missed